.mergify.yml

```diff
@@ -1,3 +1,30 @@
+queue_rules:
+  - name: default
+    conditions:
+      # Conditions to get out of the queue (= merged)
+      - base~=master|integ-tests
+      - "#approved-reviews-by>=1"
+      - approved-reviews-by=@aws-actions/aws-ecs-devx
+      - -approved-reviews-by~=author
+      - status-success=Run Unit Tests
+      - status-success=Semantic Pull Request
```

Comment:
Now that I think I should only keep these 2 lines for queuing PR right? And not the whole set of condition that are defined for merging the PR. 

What do you think @iamhopaul123 @Lou1415926

Reply:
I don't see the reason why we want to remove the other conditions 🤔 Do we not need them any more?